ID.me WebVerify SDK/IDmeWebVerifyKeychainData.m

```diff
@@ -0,0 +1,106 @@
+//
+//  IDmeWebVerifyKeychainData.m
+//  WebVerifySample
+//
+//  Created by Mathias Claassen on 12/9/16.
+//  Copyright © 2016 ID.me, Inc. All rights reserved.
+//
+
+#import "IDmeWebVerifyKeychainData.h"
+#import <SAMKeychain/SAMKeychain.h>
+
+/// Keychain Constants
+#define IDME_KEYCHAIN_DATA_ACCOUNT              @"IDME_KEYCHAIN_DATA"
+#define IDME_EXPIRATION_DATE                    @"IDME_EXPIRATION_DATE"
+#define IDME_REFRESH_TOKEN                      @"IDME_REFRESH_TOKEN"
```

Comment:
Everything is implemented so that a refresh token can be easily supported in the future. Do you want us to remove everything related to refresh tokens?